src/document/mod.rs

```diff
@@ -40,17 +43,16 @@ impl ExplicitGrant {
 /// Parameters that can be provided when encrypting a new document.
 ///
 /// Document IDs must be unique to the segment. If no ID is provided, one will be generated for it.
-/// If no document name is provided, the document's name will be left empty.
+/// If no name is provided, the document's name will be left empty.
 ///
-/// For default parameters, use `DocumentEncryptOpts::default()`.
+/// Default values are provided with `DocumentEncryptOpts::default()`.
```

Comment:
I think it would be nice to link to the Default impl here as that is where we describe what the defaults are.

Reply:
Good point. Did that here and in the arguments for `document_encrypt`/`document_encrypt_unmanaged`